src/components/CreateMenu.js

```diff
@@ -82,23 +79,13 @@ class CreateMenu extends PureComponent {
                 anchorPosition={styles.createMenuPosition}
             >
                 {menuItemData.map(({icon, text, onPress}) => (
-                    <Pressable
-                        key={text}
-                        onPress={onPress}
-                        style={({hovered}) => ([
-                            styles.createMenuItem,
-                            hovered && {backgroundColor: themeColors.buttonHoveredBG},
-                        ])}
-                    >
-                        <View style={styles.createMenuIcon}>
-                            <Icon src={icon} />
-                        </View>
-                        <View style={styles.justifyContentCenter}>
-                            <Text style={[styles.createMenuText, styles.ml3]}>
-                                {text}
-                            </Text>
-                        </View>
-                    </Pressable>
+                    <View style={styles.ph5} key={text}>
```

Comment:
I think we want the padding-horizontal to be within the menu row itself, so that this way the gray hover area extends to the full width of the rightDocked sidepane:
![image](https://user-images.githubusercontent.com/2319350/109936650-697f7500-7cce-11eb-890b-fb096fd30c65.png)


Let me know if that makes sense.

Reply:
It looks like that also broke the create menu hover too:
![image](https://user-images.githubusercontent.com/2319350/109936736-8025cc00-7cce-11eb-89d3-ba1f425d118f.png)

Here is what it currently looks like:
![image](https://user-images.githubusercontent.com/2319350/109936788-8ae06100-7cce-11eb-9de5-1efe5b6533fa.png)